cookbooks/aws-parallelcluster-slurm/libraries/update.rb

```diff
@@ -53,11 +53,18 @@ def is_compute_node_bootstrap_timeout_updated?(previous_config, config)
   evaluate_compute_bootstrap_timeout(previous_config) != evaluate_compute_bootstrap_timeout(config)
 end
 
-def is_slurm_database_updated?
+def config_parameter_changed?(param)
```

Comment:
Optional: It can be used for other functions in the file e.g. `are_bulk_custom_slurm_settings_updated`